packages/vue/src/components/organisms/SfStoreLocator/_internal/SfStore.vue

```diff
@@ -6,75 +6,73 @@
         <span>{{ distance }}km</span> away from you
       </div>
     </slot>
-    <SfButton
-      class="sf-store__btn sf-button--full-width sf-button--pure"
-      @click="centerOn(latlng)"
-    >
-      <div class="sf-store__media">
-        <!-- @slot Use this slot to show media elements -->
-        <slot name="media">
-          <SfImage
-            :src="picture"
-            :alt="`${name} picture`"
-            :width="82"
-            :height="112"
-          />
-        </slot>
-      </div>
-      <div class="sf-store__info">
-        <div class="sf-store__heading">
-          <!-- @slot Use this slot to show heading -->
-          <slot name="heading">
-            <div class="sf-store__name">
-              {{ name }}
-            </div>
-          </slot>
-        </div>
-        <!-- @slot This is the default slot of the component, placed on the right of the picture -->
-        <slot>
-          <div v-if="address" class="sf-store__address">
-            {{ address }}
+    <div :class="{ 'sf-store__media': picture }" @click="centerOn(latlng)">
+      <!-- @slot Use this slot to show media elements -->
+      <slot name="media">
+        <SfImage
+          v-if="picture"
+          :src="picture"
+          :alt="`${name} picture`"
+          :width="82"
```

Comment:
`width` should not be a fixed manual code. same with `height`. Maybe improve it?

Reply:
yes it is :)